docs/source/solving-interface.rst

```diff
@@ -728,18 +732,21 @@ solving with.
 Default solver options
 ~~~~~~~~~~~~~~~~~~~~~~
 
-If no parameters are passed to a solve call, we use, in most cases,
+
+If no parameters are passed to a ``solve`` call, we use, in most cases,
 the defaults that PETSc supplies for solving the linear or nonlinear
-system.  We describe the most commonly modified options (along with
-their defaults in Firedrake) here.  For linear variational solves we
+system. We describe the most commonly modified options (along with
+their defaults in Firedrake) here. For linear variational solves we
 use:
 
-* ``ksp_type``: GMRES, with a restart (``ksp_gmres_restart``) of 30
-* ``ksp_rtol``: 1e-7
-* ``ksp_atol``: 1e-50
-* ``ksp_divtol`` 1e4
-* ``ksp_max_it``: 10000
-* ``pc_type``: ILU (Jacobi preconditioning for mixed problems)
+* ``mat_type``: ``aij``
+* ``ksp_type``: ``preonly``
+* ``ksp_rtol``: ``1e-7``
+* ``ksp_atol``: ``1e-50``
+* ``ksp_divtol``: ``1e4``
+* ``ksp_max_it``: ``10000``
```

Comment:
These options are no longer set in the default for a linear variational solver
```suggestion
```

Reply:
Also, please make sure the rest of the file is up to date. The options in 753-763 should have:
```
{'mat_type': 'aij',
 'ksp_type': 'preonly',
 'ksp_rtol': 1e-05,
 'pc_type': 'lu',
 'pc_factor_mat_solver_type': 'mumps',
 'pc_factor_mat_mumps_icntl_14': 200,
 'snes_type': 'newtonls',
 'snes_linesearch_type': 'basic'}
```